### PR TITLE
Add AMB monitor e2e tests

### LINKED ISSUE #216: extend e2e monitor testing to use AMB
This issue has been created to address work needed to extend e2e monitor testing to use environment with AMB

### PR DESCRIPTION
**Pull request by patitonar**
closes #216

### CODE REVIEW

File: monitor-e2e/test/amb.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    it('executeAffirmations', () => assert(typeof data.executeAffirmations === 'object'))
+  })
+  describe('changing state of contracts', () => {
+    let data
+
+    before(async () => {
+      assert((await axios.get(`${baseUrl}/validators`)).data.validatorsMatch === true)
+    })
+
+    it('should change fromForeignToHomeDiff', async () => {
```
Comment (akolotov) [inline monitor-e2e/test/amb.js]: Should we extend the test to verify catching the change in case of the Homo-to-Foreign direction?

File: monitor/index.js [hunk 2]
```diff
@@ -77,10 +77,10 @@ app.get('/eventsStats', async (req, res, next) => {
   try {
     const results = await readFile('./responses/eventsStats.json')
     results.ok =
-      results.onlyInHomeDeposits.length === 0 &&
-      results.onlyInForeignDeposits.length === 0 &&
-      results.onlyInHomeWithdrawals.length === 0 &&
-      results.onlyInForeignWithdrawals.length === 0
+      (results.onlyInHomeDeposits || results.home.deliveredMsgNotProcessedInForeign).length === 0 &&
```
Comment (akolotov) [inline monitor/index.js]: could you clarify why these changes are needed now?

Comment (patitonar) [reply monitor/index.js]: This was a bug found while adding the e2e tests. For AMB `results.onlyInHomeDeposits` does not exists, so `results.onlyInHomeDeposits.length` was throwing an error

File: monitor-e2e/test/amb.js (hunk 1 above)
Comment (patitonar) [reply monitor-e2e/test/amb.js]: Yes, thanks, added the test https://github.com/poanetwork/tokenbridge/pull/231/commits/9ee0db3fba07aa57cf6fdf56834680f37fa7824d

File: monitor/index.js (hunk 2 above)
Comment (akolotov) [reply monitor/index.js]: thanks
